ddtrace/llmobs/_writer.py

```diff
@@ -217,11 +204,15 @@ def __init__(self):
         super(LLMObsEventClient, self).__init__(encoder)
 
 
+class LLMObsEventAgentlessEventClient(LLMObsEventClient):
```

Comment:
Can we rename this to `LLMObsAgentlessEventClient`? The extra `event` seems unnecessary